runtime/src/accounts_db.rs

```diff
@@ -629,10 +629,44 @@ impl AccountsDB {
         false
     }
 
-    // Purge zero lamport accounts for garbage collection purposes
+    // Reclaim older states of rooted non-zero lamport accounts as a general
+    // AccountsDB bloat mitigation and preprocess for better zero-lamport purging.
+    fn purge_old_normal_accounts(&self, purges: &HashMap<Pubkey, Vec<(Slot, AccountInfo)>>) {
```

Comment:
Can we make just one more rename for this function. Maybe something like `clean_old_account_roots`

Reply:
Nice catch!

Done: 6c3a6967f1dbe31643f77f9cdaaaaf14eded745c.

I persisted ending with `..._accounts` to avoid confusion with `AccountsIndex::clean_dead_slot`.

The former removes accounts and the later does actual slots. I think `root` as a noun connotes a (root) slot.